django/src/pages/templates/extends/base.html

```diff
@@ -139,6 +140,13 @@ <h6>Users Online</h6>
 		})
 	}
 
+	document.querySelectorAll('[data-route]').forEach((element) => {
```

Comment:
this was just moved elsewhere in main code you need to merge main

Reply:
i did merge main

Reply:
Well this is suppose to be in router.js now

Reply:
addressed in : [438d1b0](https://github.com/TheSecretOrganization/ft_transcendence/pull/247/commits/438d1b00e69532d81797f333de11c94f98e67121)